android/java/res/values-night/brave_colors.xml

```diff
@@ -51,6 +51,14 @@
 
     <!-- Brave wallet 2.0 colors -->
     <color name="brave_wallet_dn_panel_bg">@color/wallet_bg</color>
+    <color name="brave_wallet_semantic_button_disabled">#4D687B85</color>
```

Comment:
can we use nala color names here ?

Reply:
Color name updated using Nala naming convention. Done in commit 👉 [`0472a10` (#23338)](https://github.com/brave/brave-core/pull/23338/commits/0472a10d76e8db6a13ffa5079680c11d02c9fe77)